docs/src/markdown/about/changelog.md

```diff
@@ -1,25 +1,25 @@
 # Changelog
 
-## 10.14.3
+## [10.14.3](https://github.com/facelessuser/pymdown-extensions/releases/tag/10.14.3)
```

Comment:
This change is necessary to ensure `ffurrer2/extract-release-notes` works correctly.

https://github.com/ffurrer2/extract-release-notes

Reply:
May address 603204830d6c71d164780a85b1c1633fbd80d1a3 into a new PR.